azure/functions/_cosmosdb.py

```diff
@@ -27,6 +27,11 @@ def to_json(self) -> str:
         """Return the JSON representation of the document."""
         return json.dumps(dict(self))
 
+    def to_dict(self) -> dict:
+        """Return the document as a dict - directly using self would also work
+        as Document is ``UserDict`` subclass and behave like dict"""
+        return dict(self)
```

Comment:
Do we need to add a test for this?

Reply:
We don't have direct way of testing `to_json`, `to_dict` and other library methods. This is actually raising that we should have unittests for them as well.